examples/nextjs-full-stack/README.md

```diff
@@ -1,8 +1,10 @@
-# Full-stack Miro app example
 
-This sample app demonstrates the use of the Miro REST API and Web SDK. After configuring your project outlined in the Prerequisites section, this app allows you to authenticate yourself with your Miro account, and use the related access tokens to make REST API calls via the Miro Developer Platform.
+## About the app
+
+This full-stack sample app shows how to build a Next.js application that uploads a camera image to a Miro board by using the Miro Web SDK and Miro REST API. This app demonstrates the use of the Miro REST API and Web SDK. After configuring your project outlined in the Prerequisites section, this app allows you to authenticate yourself with your Miro account, and use the related access tokens to make REST API calls via the Miro Developer Platform.
```

Comment:
Why do we use `sample` in favour of `example`?

Reply:
good point - should be example not sample here - we should get rid of example or sample in all descriptions though coz itz redundant will check and update all instances - thanks!